cdap-ui/app/features/streams/streams.less

```diff
@@ -49,6 +62,7 @@ body.state-streams-detail {
       }
 
       .results {
+        margin-top: 25px;
         div {
```

Comment:
I feel a css selector for all the divs under results css class is too generic. We should probably revisit this soon. This is the same for both datasets and streams. We should also be re-using them. Not sure why we aren't.